x-pack/plugin/security/qa/slowlog/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'elasticsearch.internal-java-rest-test'
```

Comment:
Do we need to create another QA project here? It seems we could just move `SecuritySlowLogIT` into one of the existing security QA projects.

Reply:
Thanks for reviewing! 

I created a new one because the audit log had a separate one and it felt like they're a little similar. Thinking some more about it this could also fit in with the `security-basic` tests, since it's part of the basic licence. I'll move it there.